### ISSUE TITLE
Error message in CConfig

### ISSUE THREAD

**Issue by HolQue** [test-fullautomation/robotframework-testsuitesmanagement#70]
The file
`RobotFramework_TestsuitesManagement\Config\CConfig.py`
contains the following error message:
```
logger.error("Additional properties are not allowed! \n \
Please put the additional params to 'preprocessor': { 'definitions' : {...} or 'params': { 'global': {...}")
```

But as long as "preprocessor definitions" are not a valid feature, an error message should not mention this.

**Comment by namsonx** [comment]
Hello Holger,

I created the [commit](https://github.com/test-fullautomation/robotframework-testsuitesmanagement/pull/63/commits/b7b4f311bf3f00f3fa60cfa563ee20495d32802a) for this issue.

Thank you,
Son

**Comment by test-fullautomation** [comment]
solved with #63
